library/common/jni/android_jni_utility.cc

```diff
@@ -37,7 +37,7 @@ bool is_cleartext_permitted(absl::string_view hostname) {
 void tag_socket(int ifd, int uid, int tag) {
 #if defined(__ANDROID_API__)
   JNIEnv* env = get_env();
-  jclass jcls_AndroidNetworkLibrary = find_class("org/chromium/net/AndroidNetworkLibrary");
+  jclass jcls_AndroidNetworkLibrary = find_class("org.chromium.net.AndroidNetworkLibrary");
```

Comment:
@RyanTheOptimist It turns out after all that we need `.` in here as @danzh2010 ran into a case where the use `/` was causing a crash (and `.` worked just fine)

Reply:
*boggle* :)

Reply:
it's like tabs versus spaces discussion all over again.  🙃